### ISSUE TITLE
Passbolt recovery for expired registration tokens

### ISSUE THREAD

**Issue by danpospisil** [passbolt/passbolt_api#290]
According to this: https://help.passbolt.com/faq/start/registration-token-expired when user's token expires he should be able to get a new one with the recovery function. So I go to https://DOMAIN/recover which gets redirected to https://DOMAIN/users/recover Everything seems to be working except the link received (https://DOMAIN/setup/install/USER_ID/TOKEN) in the mail is not valid and Passbolt shows:

```
The authentication token is not valid or expired.

The requested address was not found on this server. Please double check the url. Maybe the page was deleted or moved.
```
Entry in authentication_tokens exists (i see that the type is not register but recover):
```
+--------------------------------------+--------------------------------------+--------------------------------------+--------+---------------------+---------------------+---------+------+
| id                                   | token                                | user_id                              | active | created             | modified            | type    | data |
+--------------------------------------+--------------------------------------+--------------------------------------+--------+---------------------+---------------------+---------+------+
|                  ID                  |                TOKEN                 |               USER_ID                |      1 | 2018-10-29 13:54:34 | 2018-10-29 13:54:34 | recover | NULL |
+--------------------------------------+--------------------------------------+--------------------------------------+--------+---------------------+---------------------+---------+------+
```

When I manually edit the type to register it starts to work.
We run Passbolt v2.4.0 ~ Final Countdown (ce)

**Comment by stripthis** [comment]
Hi danpospisil, you are right that's a regression with v2.4. I'll take care if it.

**Comment by stripthis** [comment]
This issue is fixed with v2.5.
